com.unity.ml-agents/Runtime/Sensors/ObservationWriter.cs

```diff
@@ -122,6 +122,7 @@ internal void SetTarget(TensorProxy tensorProxy, int batchIndex, int channelOffs
         /// </summary>
         /// <param name="data"></param>
         /// <param name="writeOffset">Optional write offset.</param>
+        [Obsolete("Use AddList() for better performance")]
```

Comment:
This is in the public API so we can't remove it, but we can strongly steer users to AddList instead.